x-pack/plugins/rollup/public/crud_app/sections/job_create/job_create.js

```diff
@@ -516,9 +502,6 @@ export class JobCreateUi extends Component {
               horizontalPosition="center"
               className="rollupJobWizardPage"
             >
-              <EuiBreadcrumbs breadcrumbs={breadcrumbs} responsive={false} />
```

Comment:
If we only merge this into master then this won't be too bad, but are you sure you want to remove the old breadcrumbs when the k7design is not in use? Not much lost IMO, but just checking.

Reply:
Yep, this was a request from the design team.